python/pyspark/streaming/context.py

```diff
@@ -61,6 +68,7 @@ def __init__(
         batchDuration: Optional[int] = None,
         jssc: Optional[JavaObject] = None,
     ):
+        warnings.warn("DStream is deprecated. Migrate to Structured Streaming", FutureWarning)
```

Comment:
nit: Specify the version as it won't give the information of version automatically. `DStream is deprecated as of Spark 3.4.0`.